Jellyfin.Plugin.Anime/Providers/AniDB/Identity/AniDbTitleDownloader.cs

```diff
@@ -79,17 +80,7 @@ public async Task Load(CancellationToken cancellationToken)
         private async Task DownloadTitles(string titlesFile)
         {
             _logger.LogDebug("Downloading new AniDB titles file.");
-
-            var client = new WebClient();
-
-            await AniDbSeriesProvider.RequestLimiter.Tick().ConfigureAwait(false);
-            await Task.Delay(Plugin.Instance.Configuration.AniDB_wait_time).ConfigureAwait(false);
-            using (var stream = await client.OpenReadTaskAsync(TitlesUrl))
-            using (var unzipped = new GZipStream(stream, CompressionMode.Decompress))
-            using (var writer = File.Open(titlesFile, FileMode.Create, FileAccess.Write))
-            {
-                await unzipped.CopyToAsync(writer).ConfigureAwait(false);
-            }
```

Comment:
This is the same code as below, so I'm deduping

Reply:
Improving code quality is always appreciated :)